pkg/cmd/release-payload-controller/payload_verification_controller.go

```diff
@@ -159,6 +162,10 @@ func (c *PayloadVerificationController) sync(ctx context.Context, key string) er
 		releasePayload.Status.InformingJobResults = append(releasePayload.Status.InformingJobResults, generateJobStatus(verify))
 	}
 
+	releasePayload.Status.ReleaseCreationJobResult = v1alpha1.ReleaseCreationJobResult{
+		Status: v1alpha1.ReleaseCreationJobUnknown,
```

Comment:
isn't this set down below when you set the namespace and name?

Reply:
Yep.  Removed!